packages/manager/apps/web-domains/src/alldoms/hooks/data/useGetServiceInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from '@tanstack/react-query';
+import {
+  getAllDomainAttachedToAllDom,
+  getAllDomProperty,
+  getallDomService,
+} from '@/alldoms/data/api/web-domains';
+
+interface UseGetDatagridServiceInfoProps {
+  readonly serviceName: string;
+}
+
+export const useGetServiceInfo = ({
+  serviceName,
+}: UseGetDatagridServiceInfoProps) => {
+  return useQuery({
+    queryKey: ['serviceInfoList', serviceName],
+    queryFn: async () => {
+      const [allDomProperty, serviceInfo, domainAttached] = await Promise.all([
+        getAllDomProperty(serviceName),
+        getallDomService(serviceName),
+        getAllDomainAttachedToAllDom(serviceName),
+      ]);
```

Comment:
If you need several function in same query maybe we can use useQueries and not useQuery